pkg/aws/policies.go

```diff
@@ -123,6 +123,9 @@ var AccountRoles map[string]AccountRole = map[string]AccountRole{
 	SupportAccountRole:      {Name: "Support", Flag: "support-role-arn"},
 }
 
+var OCMUserRole = "ocm-user"
```

Comment:
Maybe capitalize this: `User` to keep it consistent with all other roles we have (still debating whether we drop the `ocm-` part of the role name)